js/adapt-contrib-tutor.js

```diff
@@ -14,23 +14,30 @@ class Tutor extends Backbone.Controller {
     });
   }
 
+  getTutorType(model) {
+    const config = (!model.get('_tutor') || (model.get('_tutor')?._isInherited ?? true))
+      ? Adapt.course.get('_tutor')
+      : model.get('_tutor');
+    if (!config) return;
+
+    return TUTOR_TYPE(config._type?.toUpperCase()).asString;
+  }
+
   onComponentViewPostRender(view) {
     const { model } = view;
     if (!model.isTypeGroup('question')) return;
-    const shouldShowFeedback = (model.get('_canShowFeedback') && model.get('_isSubmitted'));
+    const type = this.getTutorType(model);
+    if (!type) return;
+    const shouldShowFeedback = (model.get('_canShowFeedback') && model.get('_isSubmitted') && type !== TUTOR_TYPE.INLINE);
     if (!shouldShowFeedback) return;
     model.setupFeedback();
     Adapt.trigger('questionView:showFeedback', view);
   }
 
   onButtonsViewPostRender(view) {
     const { model } = view;
-    const config = (!model.get('_tutor') || (model.get('_tutor')?._isInherited ?? true))
-      ? Adapt.course.get('_tutor')
-      : model.get('_tutor');
-    if (!config) return;
-    const type = TUTOR_TYPE(config._type?.toUpperCase());
-    if (type !== TUTOR_TYPE.INLINE) return;
+    const type = this.getTutorType(model);
+    if (!type || type !== TUTOR_TYPE.INLINE) return;
```

Comment:
```suggestion
    if (type !== TUTOR_TYPE.INLINE) return;
```

not sure if `!type` condition is needed